src/Hybridauth.php

```diff
@@ -120,6 +120,24 @@ public function getAdapter($name)
 
         $adapter = isset($config['adapter']) ? $config['adapter'] : sprintf('Hybridauth\\Provider\\%s', $name);
 
+        if (! class_exists($adapter)) {
```

Comment:
no need extra space after `!`, just `if (!class_exists($adapter))`

p.s. it was old style.

Reply:
well, maybe fix the whole file? I was guided by the style that was already installed.